test-fixtures/queryProfile/07/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "description": "Query profile: foo: bar => {} => 4 profile",
+  "only": false,
+  "operationParams": {
+    "foo": "bar"
```

Comment:
Why are non-controlled parameters allowed and ok response is returned? Proposing some kind of nok response to indicate parameters are invalid.

Reply:
Nope. There is query filter that drops all unwanted query params. To get that point you have allready authenticated that you have permission to it so we return result of all OK query params. In this case there is no OK params => query all

Reply:
And if we were correct. Mongo operator should allow that unwanted param in there actually. It is not mongo operators job to filter those anymore. It should be API's job.